apps/sms/test/unit/drafts_test.js

```diff
@@ -105,72 +106,60 @@ suite('Drafts', function() {
     mocksHelper.suiteTeardown();
   });
 
-  suite('Drafts() >', function() {
-
-    test('Draft', function() {
```

Comment:
Ok, IMO such tests are only for the case when not _all_ API methods are tested, otherwise we'll catch if something is missing :)